versioned_docs/version-2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -232,3 +232,13 @@ To upgrade legacy multi-cluster apps:
 ## Limitations
 
 Dashboard apps or Rancher feature charts can't be installed using the Rancher CLI.
+
+## Known Bugs 
+
+As of now, Rancher UI displays the version in `Upgradable` column by combining the versions of the chart from all the Helm repositories installed in Rancher. 
```

Comment:
```suggestion
As of now, Rancher UI displays the version in `Upgradable` column by combining the versions of the chart from all the Helm repositories installed in Rancher. 
```

As of v2.9,0, or does this affect earlier Rancher versions?

Also: From the description that follows, I think "combining" might be too brief and suggest that it displays multiple values or aggregates them in some way. I think this needs to be rephrased. Something like, "only lists the most recent latest version from available repos, even if other available repos offer earlier latest versions of the same chart"

"Combining" here seems to mean "creates a list of available versions by combining values from multiple repositories"?